shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -422,6 +415,31 @@ void updateSemantics(ByteBuffer buffer, String[] strings) {
             }
         }
 
+        // Now that the tree is pruned, sort all the entries and set their
+        // previous ids.
+        it = mObjects.entrySet().iterator();
+        ArrayList<SemanticsObject> virtualIdOrder = new ArrayList<SemanticsObject>(mObjects.size());
+        while (it.hasNext()) {
+            Map.Entry<Integer, SemanticsObject> entry = it.next();
+            virtualIdOrder.add(entry.getValue());
+        }
+        Collections.sort(virtualIdOrder, new Comparator<SemanticsObject>() {
+            public int compare(SemanticsObject a, SemanticsObject b) {
+                if (a.sortOrder == -1 || b.sortOrder == -1)
+                    return 0;
+                return Integer.compare(a.sortOrder, b.sortOrder);
+            }
+        });
+        int previousId = View.NO_ID;
+        for (SemanticsObject current : virtualIdOrder) {
+            if (current.previous != previousId) {
+                current.previous = previousId;
+                updated.add(current.id);
```

Comment:
Do we have to add these to updated and send a `TYPE_WINDOW_CONTENT_CHANGED` even if only their ordering changed?